pkg/jsonnet/find_importers_test.go

```diff
@@ -154,6 +208,11 @@ func TestFindImportersForFiles(t *testing.T) {
 	require.NoError(t, err)
 	require.NotEmpty(t, files)
 	for _, file := range files {
+		// This project is known to be invalid (as the name suggests)
+		if strings.Contains(file, "using-deleted-stuff") {
```

Comment:
Should this be part of `findImportersTestCase`? That way you can have next to the test information if you should skip here or rather, expect an error or not.

Reply:
hmm, I don't think so. It's a sanity check. The command to find importers does not eval jsonnet (because that would be too expensive), and for that that reason, it does not necessarily throw errors on invalid jsonnet (it's not its purpose). So this eval is there to check that the test cases are actually valid